2024.html

```diff
@@ -69,7 +69,7 @@ <h3>Kudos to these websites who <!-- @@ got --> plan to get naked in 2024!</h3>
             <li>Jens Oliver Meiert: <a href="https://frontenddogma.com/">Frontend Dogma</a></li>
             <li><a href="https://villapirorum.netlify.app/">Antoine Villepreux</a></li>
             <li><a href="https://cssence.com/">Matthias Zöchling</a></li>
-            <li><a href="https://github.com/css-naked-day/css-naked-day.github.io/edit/master/2024.html"><strong>Yours?</strong></a></li>
+            <li><a href="https://divsel.neocities.org">divsel.neocities.org</a></li>
```

Comment:
Please

* add a leading slash (if only for consistency)
* use your name or, if that’s a problem, the sitename (divsel?)
* sort by the current system (alphabetically)